base/server/upgrade/11.9.0/02-EnableESTFullCMC.py

```diff
@@ -0,0 +1,47 @@
+#
+# Copyright Red Hat, Inc.
+#
+# SPDX-License-Identifier: GPL-2.0-or-later
+#
+import logging
+import os
+
+import pki
+
+logger = logging.getLogger(__name__)
+
+
+class EnableESTFullCMC(pki.server.upgrade.PKIServerUpgradeScriptlet):
+
+    def __init__(self):
+        super().__init__()
+        self.message = 'Enable EST fullcmc support'
+
+    def upgrade_subsystem(self, instance, subsystem):
+        if subsystem.name != 'est':
+            return
+
+        logger.info('Update EST backend configuration for fullcmc support')
+        self.update_est_backend_conf(instance, subsystem)
+
+    def update_est_backend_conf(self, instance, subsystem):
+        """Add fullcmc.profile configuration to EST backend.conf"""
+        backend_conf = subsystem.backend_conf
+
+        if not os.path.exists(backend_conf):
+            logger.info('EST backend.conf does not exist, skipping.')
+            return
```

Comment:
IIUC `backend.conf` is guaranteed to exist in EST, otherwise it wouldn't be a valid subsystem, so we don't really need this check.